### PR TITLE
add new resources

### PR DESCRIPTION
**Pull request by josephineroh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ A curated collection of open source fintech libraries and resources. This is not
 - [Watchman](https://github.com/moov-io/watchman) - offers search functions over numerous trade sanction lists from the United States
 
 ## Stock data
+- [GoFinance](https://github.com/aktau/gofinance?ref=fintastico.com) - Financial information retrieval and munging written in Go
```
Comment (adamdecaf) [inline README.md:69]:
```suggestion
- [GoFinance](https://github.com/aktau/gofinance) - Financial information retrieval and munging written in Go
```

File: README.md [hunk 2]
```diff
@@ -87,6 +88,7 @@ A curated collection of open source fintech libraries and resources. This is not
 - [go-finance](https://github.com/alpeb/go-finance) - Go library containing a collection of financial functions for time value of money (annuities), cash flow, interest rate conversions, bonds, and depreciation calculations
 - [Gringotts](https://github.com/aviabird/gringotts) - a simple and unified API to access dozens of different payment gateways with very different APIs, response schemas, documentation, and jargon
 - [Hyperledger Fabric](https://github.com/hyperledger/fabric) - distributed ledger framework for developing solutions and applications
+- [Omnipay](https://github.com/thephpleague/omnipay?ref=fintastico.com)- payment processing library for PHP
```
Comment (adamdecaf) [inline README.md:91]:
```suggestion
- [Omnipay](https://github.com/thephpleague/omnipay)- payment processing library for PHP
```

**Review by adamdecaf — COMMENTED** [review]
Can we remove these referral links?

**Comment by josephineroh** [comment]
> 



> Can we remove these referral links?

Done!

**Comment by adamdecaf** [comment]
Thanks!

**Pull request merged**